tests/embind_function_calling_with_pthreads.cpp

```diff
@@ -0,0 +1,31 @@
+#include <assert.h>
+
```

Comment:
remove these extra newlines (and maybe sort headers)

Reply:
I thought I'd already fixed this, but forgot to push